configure.py

```diff
@@ -384,12 +384,16 @@ def set_action_env_var(environ_cp,
 def convert_version_to_int(version):
   """Convert a version number to a integer that can be used to compare.
 
+  Version strings of the form X.YZ and X.Y.Z-xxxxx are supported. The
```

Comment:
Indeed - I think this should just be merged.

If the bazel rev number is always 3 fields, then there isn't a problem, and i've not seen any examples where it isn't.